lib/core.ps1

```diff
@@ -142,11 +142,28 @@ function set_config {
         $scoopConfig.PSObject.Properties.Remove($name)
     }
 
+    # Initialize config's change
+    Complete-ConfigChange -Name $name -Value $value
+
     # Save config with UTF8NoBOM encoding
     ConvertTo-Json $scoopConfig | Out-UTF8File -FilePath $configFile
     return $scoopConfig
 }
 
+function Complete-ConfigChange {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 0)]
+        [string]
+        $Name,
+        [Parameter(Mandatory, Position = 1)]
+        [AllowEmptyString()]
+        [string]
+        $Value
+    )
+    # Add needed config change logic here
```

Comment:
An empty function?

Reply:
Yes, empty function, for later usage.

Reply:
Umm, so we'll add it when we have usage? Right now it doesn't fit here.

Reply:
Hmm, see usage in #5851. will remove from here now